aries-site/src/examples/templates/forms/ChangePasswordExample.js

```diff
@@ -9,8 +9,30 @@ import {
   TextInput,
 } from 'grommet';
 
-import { FormContainer } from '.';
-import { passwordRulesWeak } from './formHelpers';
+const passwordRulesWeak = [
+  {
+    regexp: new RegExp('.{4,}'),
+    message: 'At least four characters',
+    status: 'error',
+  },
+  {
+    regexp: new RegExp('(?=.*?[#?!@$ %^&*-])'),
+    message: 'At least one special character or space',
+    status: 'error',
+  },
+];
+
+const FormContainer = ({ ...rest }) => {
```

Comment:
Same as above, what is ..rest here and can it be removed or what value does it hold

Reply:
Or in this case, rest may be "children" right? this might be slightly different than my above comment.

Reply:
yes the rest here is children for the `formcontainer`